cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -260,7 +207,7 @@ public int run( String[] args )
             return 10;
         }
         // Don't bother skipping if we're just trying to analyse deps.
-        else if ( new File( target.getParentFile(), ManipulationManager.MARKER_FILE ).exists() && !cmd.hasOption( 'p' ) )
+        else if ( new File( target.getParentFile(), ManipulationManager.MARKER_FILE ).exists() )
```

Comment:
The previous code also checked `!printDeps` which I guess is `!printProjectDeps` now? I assume we want to add an alias for the old option names if we change them here?

Reply:
Done